app/schemas/models/school.schema.js

```diff
@@ -0,0 +1,27 @@
+const schema = require('./../schemas')
+
+const SchoolSchema = schema.object({
+  title: 'School',
+  description: 'Represents a school in our database'
+}, {
+  district: schema.objectId({ links: [ { rel: 'extra', href: '/db/district/{($)}' } ] }),
+  city: schema.shortString(),
+  state: schema.shortString(),
+  countryName: schema.shortString(),
+  county: schema.shortString(),
+  phone: schema.shortString(),
+  ncesId: schema.shortString(),
+  students: schema.int(),
+  zip: schema.shortString(),
+  geoloc: schema.object({
+    title: 'Geolog'
+  }, {
+    lat: schema.float({ title: 'lat' }),
+    lng: schema.float({ title: 'lng' })
+  })
+})
+
+schema.extendBasicProperties(SchoolSchema, 'school')
+schema.extendNamedProperties(SchoolSchema)
```

Comment:
Similar comments here on names–this adds a `slug` property, which sometimes won't be unique. Mayhap we add name separately and forget the `slug`, or make a unique slug another way.

Reply:
I think I'll use the other approach. Instead of doing the name/slug plugin and schema, will add the `name` but keep it decoupled from any slug field.

We'll likely want to index by nces id instead of by slug as well. I'm not sure an artificially unique slug provides any utility, than indexing on nces and having a non unique name field.